test/cmd/builds.sh

```diff
@@ -147,6 +147,33 @@ os::cmd::expect_success "oc delete bc/test-buildconfig"
 echo "set build-hook: ok"
 os::test::junit::declare_suite_end
 
+os::test::junit::declare_suite_start "cmd/builds/setbuildhook"
```

Comment:
will do ... just realized it should be called cmd/builds/setbuildsecret as well :)